CHANGELOG.md

```diff
@@ -1 +1,107 @@
+* Ensure CPU quota/period updates are sent to runc [moby/moby#36030](https://github.com/moby/moby/pull/36030)
+* Gitutils: fix checking out submodules [moby/moby#35737](https://github.com/moby/moby/pull/35737)
+* Honor DOCKER_RAMDISK with containerd 1.0 [moby/moby#35957](https://github.com/moby/moby/pull/35957)
+* Make container shm parent unbindable [moby/moby#35830](https://github.com/moby/moby/pull/35830)
+* Make image (layer) downloads faster by using pigz [moby/moby#35697](https://github.com/moby/moby/pull/35697)
+* Mark docker-manifest command as experimental (cli) [docker/cli#817](https://github.com/docker/cli/pull/817)
+* Progress detail [moby/moby#35798](https://github.com/moby/moby/pull/35798)
+* Remove aliases for orchestrator [docker/cli#803](https://github.com/docker/cli/pull/803)
+* Remove deprecated integration-cli/environment functions [moby/moby#36018](https://github.com/moby/moby/pull/36018)
+* Remove: add missing eol when --force is passed [docker/cli#810](https://github.com/docker/cli/pull/810)
+* Return 400 status instead of 500 for empty volume create body [moby/moby#35965](https://github.com/moby/moby/pull/35965)
+* Show container wait error [docker/cli#755](https://github.com/docker/cli/pull/755)
+* Skip graphdriver/quota TestBlockDev if dependencies are not available [moby/moby#35902](https://github.com/moby/moby/pull/35902)
+* Small nitpick: create ErrVolumeTargetIsRoot in the volume package [moby/moby#36009](https://github.com/moby/moby/pull/36009)
+* Update doc for CopyFromContainer [moby/moby#35958](https://github.com/moby/moby/pull/35958)
+* * Upgrade the frozen images to multi-arch ones [moby/moby#35929](https://github.com/moby/moby/pull/35929)
+* Use 'config' over 'configuration file' [docker/cli#785](https://github.com/docker/cli/pull/785)
+* Use errdefs instead of string contains for checking not found [moby/moby#35959](https://github.com/moby/moby/pull/35959)
++ Add `namespace method` conver in `TestV1IDService` [moby/moby#35033](https://github.com/moby/moby/pull/35033)
++ Add helpers to create errdef errors [moby/moby#35638](https://github.com/moby/moby/pull/35638)
++ Add manifest command [docker/cli#138](https://github.com/docker/cli/pull/138)
++ Add required pkg-config for Dockerfile.simple [moby/moby#35983](https://github.com/moby/moby/pull/35983)
++ Add timeouts for volume plugin ops [moby/moby#35441](https://github.com/moby/moby/pull/35441)
+- Fix issue of filter in `docker ps` where `health=starting` returns nothing [moby/moby#35940](https://github.com/moby/moby/pull/35940)
+- Fix lint on master [docker/cli#826](https://github.com/docker/cli/pull/826)
+- Fix race with concurrent daemon startup in tests [moby/moby#35982](https://github.com/moby/moby/pull/35982)
+- Fix timeout issue of  multi-services creation on AArch64 [moby/moby#35998](https://github.com/moby/moby/pull/35998)
+* Improve swagger schema for code generation [moby/moby#35593](https://github.com/moby/moby/pull/35593)
+
+### Delete
```

Comment:
What's this?

Reply:
The `delete` section is intended to be a temporary field. 
Basically it reflects everything I would intend to delete. Having a delete fields helps to ensure that I don't delete something that would be important to include in the changelog.